nixos/modules/services/web-apps/gancio.nix

```
@@ -174,7 +174,7 @@ in
         if [[ "$USER" != ${cfg.user} ]]; then
           sudo="exec /run/wrappers/bin/sudo -u ${cfg.user}"
         fi
-        $sudo ${lib.getExe cfg.package} ''${1:--help}
+        $sudo ${lib.getExe cfg.package} ''${@:--help}
```

Should this be quoted so that options that contain space are properly forwarded?
```suggestion
        $sudo ${lib.getExe cfg.package} "''${@:--help}"
```